### DISCUSSION TITLE
School doesn't provide English documents.

### DISCUSSION THREAD

**Discussion by yikerman** [community/community#47926] [GitHub Education] [answered]
I'm receiving the following issues:

>You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
> Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?
> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
>The image you submitted does not appear to match the proof type you selected. If your original document is not in English, please take a photo of the original next to an English translation.

I'm using my Student ID as the proof. However there's no English translations. How should I deal with it?

**Comment by samus-aran** [comment, marked as answer]
Hi @xiaoyu2006 👋 

We're happy you're here! Let's see how we can assist.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. We also accept pre-translated evidence if the text is not English. Take a photo of the translated evidence alongside the original and submit.

Let us know how you get on. 👍

**Reply by yikerman** [reply]
Hi, and thanks for reaching out for me.

I’m pretty sure the image is legible. It’s just a content issue.

In my school the "student ID" is a 2-page book instead of a card. You fill in the forms and the school will stamp it as a confirmation. The book contains the school’s name, my name and expiration date. However there isn’t an English version.

What should I do? Translate it myself?

**Reply by samus-aran** [reply]
Yes @xiaoyu2006, we accept pre-translated evidence. You can translate the original copy into english and take a photo of the translated text alongside the original evidence. 

Let us know how you get on. 👍

**Reply by yikerman** [reply]
@samus-aran Thanks! My handwritten translation has passed the pre-check and I hope everything is all right 😄.

By the way I have applied the student pack once 3 years ago and I didn't encounter these troubles at that time. The improved system is a little bit more unfriendly to non-English schools 😕.
